packages/website/src/features/Router/RouterPage.tsx

```diff
@@ -0,0 +1,220 @@
+          <Link as={NextLink} href="/search">
+            package manager
+          </Link>
+          . Here, we add the <Code>--registry-priority local</Code> flag to
+          ensure we're using the version of this package that you just built,
+          regardless of what others have published.)
+        </Text>
+        <Box mb={4}>
+          <CommandPreview command="cannon sample-router-project --registry-priority local" />
+        </Box>
+        <Text mb={4}>
+          Press <Code>i</Code> to interact with the contracts in this project.
+          You'll see that the router contract exposes the functions from both
+          contracts.{' '}
+          <strong>
+            Always remember to interact with the router contract, and not the
+            dependent contracts directly.
+          </strong>
+        </Text>
+        <Heading size="md" mb={4} mt={6}>
+          Add Upgradability
+        </Heading>
+        <Text mb={4}>
+          We can also deploy a{' '}
+          <Link as={NextLink} href="/packages/transparent-upgradable-proxy">
+            transparent upgradeable proxy
+          </Link>{' '}
+          pointing at the router, such that this protocol can be upgradeable. In
+          the Cannonfile, add a setting for the admin (which will be allowed to
+          upgrade the proxy) and then provision the package which includes{' '}
+          <Link
+            href="https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/transparent/TransparentUpgradeableProxy.sol"
+            isExternal
+          >
+            the proxy contract
+          </Link>
+          :
+        </Text>
+        <Box mb={4}>
+          <CodePreview code={code2} language="toml" />
+        </Box>
+        <Text mb={4}>
+          If you alter one of your contracts, when building, Cannon will
+          automatically detect this, generate a new router, and upgrade the
+          proxy to point at it. When building an upgrade, use the{' '}
+          <Code>--upgrade-from</Code> flag to reference your existing package.{' '}
+          <strong>
+            Remember to always call functions on the proxy rather than the
+            router, and{' '}
+            <Link
+              href="https://docs.openzeppelin.com/upgrades-plugins/1.x/proxies"
+              isExternal
+            >
+              avoid storage collisions
+            </Link>{' '}
+            when upgrading.
+          </strong>
+        </Text>
+        <Text>
+          If the protocol is owned by a{' '}
+          <Link isExternal href="https://safe.global/">
+            Safe
+          </Link>
+          , you can use the{' '}
+          <Link as={NextLink} href="/deploy">
+            deployer
+          </Link>{' '}
+          to run upgrades. When your protocol no longer needs to be upgraded, it
+          can be made immutable with a call to <Code>renounceOwnership</Code> on
+          the proxy.
+        </Text>
+      </Box>
```

Comment:
might want to add a section discussing how to handle storage collisions. Obviously storage collisions are outside the scope of what cannon does for you, but for any upgradable contract architecture it shouldn't hurt to add a quick appendix or somtething with useful guides related to running proxies--including avoiding collisions.